tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
+        wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"))
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+        TestUtils.restoreWskProps(wskpropsBackup)
+        wskpropsBackup.delete()
+    }
+
     it should "show api build version" in {
-        val stdout = wsk.cli(wskprops.overrides ++ Seq("property", "get", "--apibuild")).stdout
+        val wskpropsBackup = TestUtils.backupWskProps()
+        wsk.cli(wskprops.overrides ++ Seq("property", "set"))
```

Comment:
Some of the CLI arguments, like --auth, --apihost, --apiversion and others, are global and defined against the wsk command (these flags each take an argument).  The `wsk property` command overrides these flags (these flags do not take arguments).  That's currently why a `wsk property get --apibuild --apihost x.y.z` command does not obtain the API build value from the x.y.z whisk host.  I'll need to look at this more to see if we can support both flags simultaneously.